eZ/Bundle/EzPublishCoreBundle/Command/NormalizeImagesPathsCommand.php

```diff
@@ -0,0 +1,127 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace eZ\Bundle\EzPublishCoreBundle\Command;
+
+use eZ\Publish\Core\FieldType\Image\ImageStorage\Gateway\DoctrineStorage;
+use eZ\Publish\Core\IO\FilePathNormalizerInterface;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+class NormalizeImagesPathsCommand extends Command
+{
+    const BEFORE_RUNNING_HINTS = <<<EOT
```

Comment:
CS, access modifier is required
```suggestion
    private const BEFORE_RUNNING_HINTS = <<<EOT
```